api/src/main/resources/messages_es.properties

```diff
@@ -12,10 +12,10 @@
 # Copyright (C) OpenMRS, LLC.  All Rights Reserved.
 #
 
-coreapps.title=Módulo de aplicaciones centrales
-coreapps.patientDashboard.description=Aplicación Registro de Paciente
+coreapps.title=M�dulo de aplicaciones centrales
+coreapps.patientDashboard.description=Aplicaci�n Registro de Paciente
```

Comment:
Hi @ibacher thanks for the response on this,i am closing it so as i make a better pr without all these unwanted code formats